src/data/forms.js

```diff
@@ -699,6 +701,41 @@ export const BOOST_FORMS = {
     fields: [[]],
     tx: TX.CREATE_WRAP_N_ZAP,
   },
+  SNAPSHOT: {
+    id: 'SNAPSHOT',
+    title: 'Show Snapshot Proposals',
+    subtitle:
+      'View your community&apos;s snapshot proposals directly within DAOhaus for easy access.',
+    required: ['space'],
+    fields: [
+      {
+        ...FIELD.TITLE,
+        label: 'Snapshot Space',
+        name: 'space',
+      },
+    ],
+    indicatorStates: {
+      loading: {
+        spinner: true,
+        title: 'Submitting...',
+        explorerLink: true,
+      },
+      error: {
+        icon: BiErrorCircle,
+        title:
+          "Error: No space found! Please use the .eth name in your space's url",
+        errorMessage: true,
+      },
+    },
+    nextFormHook: async ({ values }) => {
```

Comment:
I think we need to revisit this.

The legos are intended to be expressed as static content, not functions that need to execute. We should look at this together and think of a different spot to get this or way to express without having the function in the lego. Maybe if they are more like the customValidations - see rageQuit:

```
  RAGE_QUIT: {
    customValidations: ['canRagequit', 'rageQuitMinimum', 'rageQuitMax'],
    id: 'RAGE_QUIT',
    title: 'Rage Quit',
    required: [],
    tx: TX.RAGE_QUIT,
    fields: [
      [
        FIELD.RAGE_QUIT_INPUT,
        {
          ...FIELD.RAGE_QUIT_INPUT,
          htmlFor: 'loot',
          label: 'Loot to Rage',
          name: 'loot',
        },
      ],
    ],
  },
  
  and TxContext.jsx 
  handleCustomValidation
  ```
  
  
  
  then formbuilder knows to run those if they are there?

Reply:
Yes, let's take a look at this together. There are a couple things making this tricky